doc/api/buffer.md

```diff
@@ -1952,17 +1940,55 @@ end of `buf` rather than the beginning.
 ```js
 const buf = Buffer.from('buffer');
 
-console.log(buf.slice(-6, -1).toString());
+console.log(buf.subarray(-6, -1).toString());
 // Prints: buffe
-// (Equivalent to buf.slice(0, 5).)
+// (Equivalent to buf.subarray(0, 5).)
 
-console.log(buf.slice(-6, -2).toString());
+console.log(buf.subarray(-6, -2).toString());
 // Prints: buff
-// (Equivalent to buf.slice(0, 4).)
+// (Equivalent to buf.subarray(0, 4).)
 
-console.log(buf.slice(-5, -2).toString());
+console.log(buf.subarray(-5, -2).toString());
 // Prints: uff
-// (Equivalent to buf.slice(1, 4).)
+// (Equivalent to buf.subarray(1, 4).)
+```
+
+### buf.slice([start[, end]])
+<!-- YAML
+added: v0.3.0
+changes:
+  - version: v7.1.0, v6.9.2
+    pr-url: https://github.com/nodejs/node/pull/9341
+    description: Coercing the offsets to integers now handles values outside
+                 the 32-bit integer range properly.
+  - version: v7.0.0
+    pr-url: https://github.com/nodejs/node/pull/9101
+    description: All offsets are now coerced to integers before doing any
+                 calculations with them.
+-->
+
+* `start` {integer} Where the new `Buffer` will start. **Default:** `0`.
+* `end` {integer} Where the new `Buffer` will end (not inclusive).
+  **Default:** [`buf.length`][].
+* Returns: {Buffer}
+
+Returns a new `Buffer` that references the same memory as the original, but
+offset and cropped by the `start` and `end` indices.
+
+This is the same behavior as `buf.subarray()`.
+
+Note that this method is not compatible with the `Uint8Array#slice()`, which is a superclass of `Buffer`. If a copy of slice is needed, use `Uint8Array.prototype.slice` directly.
```

Comment:
Can you wrap lines at 80 characters (I think `make lint` should complain otherwise), and avoid the `#` notation, as I believe we do not use it elsewhere in the documentation either?

```suggestion
Note that this method is not compatible with the `Uint8Array.prototype.slice()`,
which is a superclass of `Buffer`. If a copy of the slice is needed, use
`Uint8Array.prototype.slice` directly.
```

Reply:
Thanks for telling me with `make lint` 😄